lib/absinthe/type/built_ins/introspection.ex

```diff
@@ -84,8 +84,19 @@ defmodule Absinthe.Type.BuiltIns.Introspection do
       :field,
       :fragment_definition,
       :fragment_spread,
-      :inline_fragment
-      # TODO: Schema definitions to support Schema input
+      :inline_fragment,
+      :operation_definition,
```

Comment:
Is this correct? I don't think it is...

directive locations are defined here: https://graphql.github.io/graphql-spec/draft/#sec-Schema-Introspection

Reply:
Agreed. I removed this in a commit, pushed.